tests/ia/pages/alertsPage.js

```diff
@@ -2,27 +2,33 @@ const { I } = inject();
 const assert = require('assert');
 
 const alertRow = (alertName) => `//tr[td[contains(., "${alertName}")]]`;
+const details = locate('$alert-details-wrapper');
 
 module.exports = {
   url: 'graph/integrated-alerting/alerts',
   columnHeaders: ['Name', 'Severity', 'State', 'Labels', 'Active Since', 'Last Notified', 'Actions'],
   elements: {
     noData: '$table-no-data',
     alertRow: (alertName) => alertRow(alertName),
-    labelsCell: (alertName) => `${alertRow(alertName)}/td[4]//span`,
+    labelsCell: (alertName, text) => locate(`${alertRow(alertName)}/td[4]`).find('$chip').withText(text),
```

Comment:
why did you change `labelsCell` locator? it is not `labelsCell` anymore in your locator. It is a `primaryLabel` locator